docs/content/en/docs/Advanced/partition_encryption.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Encrypting User Data with Kairos"
+linkTitle: "Encrypting User Data with Kairos"
+weight: 5
+description: >
+    This section describes how to encrypt partition with LUKS in Kairos.
+---
+
+{{% alert title="Note" color="warning" %}}
```

Comment:
MAybe we should have a code block to mark when X feature was available from, like in goreleaser docs, I really like that stuff and its reusable:

![image](https://user-images.githubusercontent.com/1447686/214565251-efbb3117-1cd9-494d-a1a4-bb6f8cfee6cc.png)